duckduckgo/iOS
# https upgrade

Reviewer: Mia or Caine
Asana: https://app.asana.com/0/414235014887631/488055746862775
CC:

**Description**:

Perform a https upgrade for a given list of domains.

**Steps to test this PR**:
1. Visit http://booking.com observe the final URL being https (this is a wildcard domain match)
1. Visit http://www.businessinsider.com and observe final URL being https (this is a simple domain match)

Check the logs to see an 'upgraded url' message.

###### Reviewer Checklist:


###### PR DRI Checklist:
